themes/digital.gov/layouts/shortcodes/img.html

```diff
@@ -58,25 +60,13 @@
         {{- end -}}"
       {{- end -}}
       {{- if eq $prefix $thisuid -}}
-        srcset="
-        {{- if ge $imgWidth 200 -}}
-          {{- $imgBaseCDN -}}_bu.jpg 48w,
-          {{- if ge $imgWidth 400 -}}
-            {{- if ge $imgWidth 800 -}}
-              {{- if ge $imgWidth 1200 -}}
-                {{- $imgBaseCDN -}}_w1200.{{- $imgExt }} 1200w,
-              {{- end -}}
-              {{- $imgBaseCDN -}}_w800.{{- $imgExt }} 800w,
-            {{- end -}}
-            {{- $imgBaseCDN -}}_w400.{{- $imgExt }} 400w,
-          {{- end -}}
-          {{- $imgBaseCDN -}}_w200.{{- $imgExt }} 200w
-        {{- else -}}
-          {{- $imgBaseCDN -}}.{{- $imgExt }} {{ $imgWidth -}}w
-        {{- end -}}
-        "
+        srcset=" {{- $imgBaseCDN -}}_bu.jpg 48w,
+        {{- $imgBaseCDN -}}_w200.{{- $imgExt }} 200w,
+        {{- $imgBaseCDN -}}_w400.{{- $imgExt }} 400w,
+        {{- $imgBaseCDN -}}_w800.{{- $imgExt }} 800w,
+        {{- $imgBaseCDN -}}_w1200.{{- $imgExt }} 1200w "
       {{- end -}}
-      sizes="(max-width: 800px) 100vw, 400px"
+      sizes="(max-width: 400px) 100vw, (max-width: 800px) 100vw, (max-width: 1200px) 100vw, 50vw"
```

Comment:
determines the width of image based on screen size

Reply:
We should change the width size to `100vw`:

```html
sizes="(max-width: 400px) 100vw, (max-width: 800px) 100vw, (max-width: 1200px) 100vw, 50vw"
```

Setting anything over to `100vw` will still render at 100% the viewport size, see this [note on MDN docs](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#sect2).